sys/net/include/rpl.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2013, 2014  INRIA.
+ * Copyright (C) 2013 - 2015  INRIA.
```

Comment:
This PR is just a major refactoring, there were no additional functionalities added by my side. Is it right to add myself to the Copyright section?